parsec/scheduling.c

```diff
@@ -624,7 +631,7 @@ static int __parsec_taskpool_wait( parsec_taskpool_t* tp, parsec_execution_strea
         /* If there is a single process run and the main thread is in charge of
          * progressing the communications we need to make sure the comm engine
          * is ready for primetime. */
-        remote_dep_mpi_on(tp->context);
+        parsec_ce.enable(&parsec_ce);
```

Comment:
Similar comment and call to `enable` is at https://github.com/ICLDisco/parsec/pull/521/files#diff-f3c94ef414b137da235f2a72ab4ba0b93d19b740d990ed9339d3d79881c6675bR729 but there we also `remote_dep_ce_reconfigure(parsec_context)` and `parsec_remote_dep_reconfigure(parsec_context)`. On purpose? Why?

Reply:
singleton case, we only use MPI for local reshaping so we don't need the entire infrastructure (there will be no AM and no data transfer).